AND-81: Cleanup existing KTX (#51)
RedMadRobot/redmadrobot-android-ktx: fragment-args-ktx/CHANGELOG.md

## Unreleased

- **Bundle**: Removed nullable accessors with default parameter

> - **Bundle**: Bundle extensions for serializable and parcelizable now is inline-functions

Reviewer: ```suggestion
- **Bundle**: Accessors for `Serializable` and `Parcelizable` are turned to inline-functions
```
